### PR TITLE
add support for rspec 3.6.0

### PR DESCRIPTION
**Pull request by dthorsen**
This should address issue #75

### CODE REVIEW

File: gemfiles/rspec_3.5.gemfile [hunk 1]
```diff
@@ -4,4 +4,4 @@ source "https://rubygems.org"
 
 gem "rspec", "~> 3.5.0"
 
-gemspec :path => "../"
```
Comment (michaelglass) [inline gemfiles/rspec_3.5.gemfile]: please revert this change

File: Appraisals [hunk 2]
```diff
@@ -9,3 +9,7 @@ end
 appraise 'rspec-3.5' do
   gem 'rspec', '~> 3.5.0'
 end
+
+appraise 'rspec-3.6' do
```
Comment (michaelglass) [inline Appraisals]: 😻

File: gemfiles/rspec_3.3.gemfile [hunk 3]
```diff
@@ -4,4 +4,4 @@ source "https://rubygems.org"
 
 gem "rspec", "~> 3.3.0"
 
-gemspec :path => "../"
+gemspec path: "../"
```
Comment (michaelglass) [inline gemfiles/rspec_3.3.gemfile]: plz revert this change

File: gemfiles/rspec_3.4.gemfile [hunk 4]
```diff
@@ -4,4 +4,4 @@ source "https://rubygems.org"
 
 gem "rspec", "~> 3.4.0"
 
-gemspec :path => "../"
+gemspec path: "../"
```
Comment (michaelglass) [inline gemfiles/rspec_3.4.gemfile]: plz revert this change

**Comment by michaelglass** [comment]
didn't realize path changes came from appraisal. Sorry for the revert request

**Comment by michaelglass** [comment]
released: https://rubygems.org/gems/rspec-retry/versions/0.5.4

**Comment by dthorsen** [comment]
Thanks! No problem on the revert request.
